interfaces/builtin/locale_control.go

```diff
@@ -33,6 +33,34 @@ const localeControlBaseDeclarationSlots = `
 const localeControlConnectedPlugAppArmor = `
 # Description: Can manage locales directly separate from 'config ubuntu-core'.
 
+#include <abstractions/dbus-strict>
+
+# Introspection of org.freedesktop.locale1
+dbus (send)
+	bus=system
+	path=/org/freedesktop/locale1
+	interface=org.freedesktop.DBus.Introspectable
+	member=Introspect,
+# Properties of org.freedesktop.locale1
+dbus (send)
+	bus=system
+	path=/org/freedesktop/locale1
+	interface=org.freedesktop.DBus.Properties
+	member=Get{,All},
+# do not use peer=(label=unconfined) here since this is DBus activated
+dbus (send)
+	bus=system
+	path=/org/freedesktop/locale1
+	interface=org.freedesktop.locale1
+	member={SetLocale,SetX11Keyboard,SetVConsoleKeyboard}
+	peer=(name=org.freedesktop.locale1),
+# Receive Accounts property changed events
```

Comment:
this comment seems wrong

Reply:
Mmm... but then, it is also wrong in time_control, timeserver_control, timezone_control and upower_observe...

Reply:
(the magic of Copy&Paste :-D )